specs/core/0_beacon-chain.md

```diff
@@ -1587,41 +1608,36 @@ _Note_: If there are skipped slots between a block and its parent block, run the
 
 ### Per-slot processing
 
-Below are the processing steps that happen at every slot.
-
-#### Slot
+Below are the processing steps that happen at every `slot >= GENESIS_SLOT`.
```

Comment:
I think it's actually `slot > GENESIS_SLOT` as the genesis state is the post state of the genesis block.

```suggestion
Below are the processing steps that happen at every `slot > GENESIS_SLOT`.
```

Reply:
After thinking about it, I think the current genesis approach is fine, and you're right that we want `>` (as opposed to `>=`).